src/transformers/modeling_utils.py

```diff
@@ -4190,6 +4190,18 @@ def warn_if_padding_and_no_attention_mask(self, input_ids, attention_mask):
 
             logger.warning_once(warn_string)
 
+    @property
+    def _is_quantized_training_enabled(self):
+        logger.warning(
+            "`_is_quantized_training_enabled` is going to be deprecated in a future version. Please use `model.hf_quantizer.is_trainable` instead",
```

Comment:
Message should have a specific version listed here for deprecation

Reply:
ok perfect, I will set it on the next 2 minor releases